staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -1534,7 +1532,7 @@ type KubeVirtStatus struct {
 	ObservedDeploymentID                    string              `json:"observedDeploymentID,omitempty" optional:"true"`
 	OutdatedVirtualMachineInstanceWorkloads *int                `json:"outdatedVirtualMachineInstanceWorkloads,omitempty" optional:"true"`
 	// +listType=atomic
-	Generations []operatorsv1.GenerationStatus `json:"generations,omitempty" optional:"true"`
+	Generations []GenerationStatus `json:"generations,omitempty" optional:"true"`
```

Comment:
fyi, We're considering go with a changesimilar to this earlier due to an relase-0.41 blocker, see https://github.com/kubevirt/kubevirt/pull/5609

Reply:
Thanks